pages/api/Schema/index.ts

```diff
@@ -1,10 +1,10 @@
 import { buildFederatedSchema } from '@apollo/federation';
-import ContactFiltersResolvers from './ContactFilters/resolvers';
-import ContactFiltersTypeDefs from './ContactFilters/typeDefs';
-import ExportContactsResolvers from './ExportContacts/resolvers';
-import ExportContactsTypeDefs from './ExportContacts/typeDefs';
-import TaskAnalyticsResolvers from './TaskAnalytics/resolvers';
-import TaskAnalyticsTypeDefs from './TaskAnalytics/typeDefs';
+import { ContactFiltersResolvers } from './ContactFilters/resolvers';
+import ContactFiltersTypeDefs from './ContactFilters/contactFilters.graphql';
```

Comment:
Only bad thing is, ```.graphql``` files don't seem to be able to be auto imported in. Had to manually type out the file name. But I guess that's better than an additional file.

Reply:
Hmm. You could use the ts files with the typeDefs instead of the `.graphql` files if that's better. Codegen should be able to read from ts files with `gql` template literals.

Reply:
I feel like codegen wasn't working with just the typedef ts files, but once again that could be a different issue I was experiencing. I'll give it another try and see what works.

Reply:
@OzzieOrca Tried switching to just a typeDefs ts file and deleting the ```.graphql``` file. But once I did that it doesn't generate the types.

Reply:
Hmm. What file pattern was in `schema` for codegen?

Reply:
I changed both ```codegen-server.yml``` and ```codegen.yml``` to use ```schema: ./pages/api/Schema/**/*.ts``` instead of ```schema: ./pages/api/Schema/**/*.graphql```, but no luck.

Reply:
`schema: ./pages/api/Schema/**/typeDefs.ts` seems to work if you want to go that route. I'm still not sure which is the better format. `.graphql` files are nice.

I thought I might just be able to generate the gql tag version with https://www.graphql-code-generator.com/docs/plugins/typescript-document-nodes but it didn't output anything new other than the gql import. Maybe it only works with operations...